src/IJ_mv/IJMatrix_parcsr.c

```diff
@@ -473,6 +475,7 @@ hypre_IJMatrixGetValuesParCSR( hypre_IJMatrix *matrix,
       col_map_offd = hypre_ParCSRMatrixColMapOffd(par_matrix);
    }
 
+   /* WM: question - what does rows < 0 signify? Do I need to add row_indexes here? */
```

Comment:
@ulrikeyang I'm curious about this as well. What is the use case for `nrows < 0` here?

Reply:
I'm pretty sure this means "get all rows".  I think we've also used something similar to "get all cols" for a given row.  Looking through the code here, that's what seems to be happening.

Reply:
Thanks, Rob! In that case, it seems `row_indexes` is not needed here